Signal/src/view controllers/DebugUITableViewController.m

```diff
@@ -0,0 +1,271 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "DebugUITableViewController.h"
+#import "ThreadUtil.h"
+#import "Environment.h"
+#import <SignalServiceKit/TSThread.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class OWSTableItem;
+@class OWSTableSection;
+
+@interface OWSTableContents : NSObject
+
+@property (nonatomic) NSString *title;
+@property (nonatomic) NSMutableArray *sections;
+
+@end
+
+#pragma mark -
+
+@implementation OWSTableContents
+
+- (void)addSection:(OWSTableSection *)section {
+    OWSAssert(section);
+    
+    if (!_sections) {
```

Comment:
nit: better to initialize `_sections` in `init` so we can actually treat it as nonnull property.